src/components/Layout/DrawerNavigation.tsx

```diff
@@ -0,0 +1,196 @@
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    position: "relative"
+  },
+  navLink: {
+    flexDirection: "row",
+    alignItems: "center",
+    paddingHorizontal: size(4),
+    paddingVertical: size(2.5),
+    borderBottomColor: color("grey", 20),
+    borderBottomWidth: 1
+  },
+  navLinkText: {
+    color: color("blue", 50),
+    fontFamily: "brand-bold"
+  },
+  bottomNavContainerLink: {
+    flexDirection: "row",
+    alignItems: "center",
+    paddingHorizontal: size(4),
+    paddingVertical: size(1)
+  },
+  bottomNavContainerText: {
+    color: color("blue", 50),
+    fontSize: fontSize(-1)
+  }
+});
+
+interface BottomNavigationLink {
+  onPress: () => void;
+}
+
+export const BottomNavigationLink: FunctionComponent<BottomNavigationLink> = ({
+  children,
+  onPress
+}) => {
+  return (
+    <View style={styles.bottomNavContainerLink}>
+      <TouchableOpacity onPress={onPress}>
+        <AppText style={styles.bottomNavContainerText}>{children}</AppText>
+      </TouchableOpacity>
+    </View>
+  );
+};
+
+export const DrawerButton: FunctionComponent<DrawerButtons> = ({
+  icon,
+  label,
+  onPress
+}) => {
+  return (
+    <TouchableOpacity onPress={onPress}>
+      <View style={styles.navLink}>
+        <MaterialCommunityIcons
+          name={icon}
+          size={size(2.5)}
+          color={color("blue", 50)}
+          style={{ marginRight: size(2) }}
+        />
+        <AppText style={styles.navLinkText}>{label}</AppText>
+      </View>
+    </TouchableOpacity>
+  );
+};
+
+export const DrawerNavigationComponent: FunctionComponent<DrawerContentComponentProps> = ({
+  navigation
+}) => {
+  const { logout } = useLogout();
+  const showHelpModal = useContext(HelpModalContext);
+  const { drawerButtons } = useDrawerContext();
+  const handleLogout = useCallback((): void => {
+    logout(navigation.dispatch);
+  }, [logout, navigation.dispatch]);
+
+  const onPressLogout = (): void => {
+    Alert.alert(
+      "Are you sure you want to log out?",
+      "Stay logged in to change your scanning location.",
+      [
+        {
+          text: "Cancel"
+        },
+        {
+          text: "Logout",
+          onPress: handleLogout,
+          style: "destructive"
+        }
+      ],
+      { cancelable: false }
+    );
+  };
+
+  const onPressCloseDrawer = (): void => {
+    navigation.dispatch(DrawerActions.closeDrawer());
+  };
+
+  return (
+    <SafeAreaView
+      style={styles.container}
+      forceInset={{ top: "always", horizontal: "never" }}
+    >
+      <TouchableOpacity
+        onPress={onPressCloseDrawer}
+        style={{
+          position: "absolute",
+          right: size(3),
+          top: size(5),
+          padding: size(1)
+        }}
+      >
+        <MaterialCommunityIcons
+          name="close"
+          size={size(3)}
+          color={color("blue", 50)}
+        />
+      </TouchableOpacity>
+      <View
+        style={{
+          marginTop: size(9),
```

Comment:
Once the above is done, we can reduce this to maybe `size(3)` or `size(2)`, gotta see how it looks